cli/src/bin/phylum.rs

```diff
@@ -104,41 +99,33 @@ async fn handle_commands() -> CommandResult {
         None => env_logger::Builder::from_env(Env::default().default_filter_or("warn")).init(),
     };
 
-    let settings_path = config::get_home_settings_path()?;
-    let config_path = matches
-        .get_one::<String>("config")
-        .and_then(|config_path| shellexpand::env(config_path).ok())
-        .map(|config_path| PathBuf::from(config_path.to_string()))
-        .unwrap_or(settings_path);
-
-    log::debug!("Reading config from {}", config_path.to_string_lossy());
-    let mut config: Config = config::read_configuration(&config_path).map_err(|err| {
-        anyhow!("Failed to read configuration at `{}`: {}", config_path.to_string_lossy(), err)
-    })?;
-    config.set_ignore_certs_cli(matches.get_flag("no-check-certificate"));
+    let mut config = config::load_config(&matches)?;
 
+    config.set_ignore_certs_cli(matches.get_flag("no-check-certificate"));
     if config.ignore_certs() {
         log::warn!("Ignoring TLS server certificate verification per user request.");
     }
 
     // We initialize these value here, for later use by the PhylumApi object.
     let timeout = matches.get_one::<String>("timeout").and_then(|t| t.parse::<u64>().ok());
 
-    // Check for updates if enabled and if we haven't explicitly invoked `update`.
-    if cfg!(feature = "selfmanage") && matches.subcommand_matches("update").is_none() {
-        check_for_updates(&mut config, &config_path).await?;
+    let (subcommand, sub_matches) = matches.subcommand().unwrap();
+
+    // Check for updates if enabled and if we haven't explicitly invoked `update` or
+    // `parse-sandboxed`.
+    if cfg!(feature = "selfmanage") && subcommand != "update" && subcommand != "parse-sandboxed" {
```

Comment:
Decided *against* a separate flag for update checks since we're already matching for subcommands to avoid printing messages for the update command itself. If there is a real reason for a dedicated flag, we can always change it in the future.